### ISSUE TITLE
Anchor has parameter initializer in index.d.ts

### ISSUE THREAD

**Issue by lokidokicoki** [exceljs/exceljs#801]
I get this error with TypeScript 3.2.4 with exceljs@1.9.1:

```
Error: /Users/loki/work/app/node_modules/exceljs/index.d.ts(788)
 A parameter initializer is only allowed in a function or constructor implementation.
```

From the index.d.ts

```javascript
export class Anchor implements IAnchor{
    col: number;
    nativeCol: number;
    nativeColOff: number;
    nativeRow: number;
    nativeRowOff: number;
    row: number;

    private readonly colWidth: number;
    private readonly rowHeight: number;
    worksheet: Worksheet;
    
    constructor(model: IAnchor|object = {});
}
```

If I remove the initialiser the code compiles

```javascript
export class Anchor implements IAnchor{
    col: number;
    nativeCol: number;
    nativeColOff: number;
    nativeRow: number;
    nativeRowOff: number;
    row: number;

    private readonly colWidth: number;
    private readonly rowHeight: number;
    worksheet: Worksheet;
    
    constructor(model: IAnchor|object);
}
```

**Comment by eseliger** [comment]
It has been fixed, but not release triggered since then :(

**Comment by eseliger** [comment]
@guyonroche can you help out?

**Comment by lokidokicoki** [comment]
Ah ok. well i look forward to the new release :)

Thanks for such a great bit of software!

**Comment by lokidokicoki** [comment]
Fixed in exceljs@1.13.0
